### PR TITLE
Manage ACL entries for private files, fix bug with copy tasks

### PR DESCRIPTION
**Pull request by drybjed**

### CODE REVIEW

File: files/usr/local/lib/pki/pki-realm [hunk 1]
```diff
@@ -776,6 +803,18 @@ generate_gnutls_rsa_private_key () {
         sed -n -i '/-----BEGIN RSA PRIVATE KEY-----/,$ p' ${key_file}.tmp
         chmod ${config['private_file_mode']} ${key_file}.tmp
         chgrp ${key_group} ${key_file}.tmp
+        if [ "${key_acl}" = "true" ] ; then
+            if type setfacl > /dev/null 2>&1 ; then
+                local private_file_acl_groups_r=( $(echo "${config['private_file_acl_groups_r']}" | tr "/" " ") )
+                if [ ${#private_file_acl_groups_r[@]} -ne 0 ] ; then
+                    for acl_group in ${private_file_acl_groups_r[@]} ; do
+                        if $(getent group ${acl_group} > /dev/null) ; then
+                            setfacl -m "g:${acl_group}:r" ${key_file}.tmp
```
Comment (ypid) [inline files/usr/local/lib/pki/pki-realm]: @drybjed Please quote variables in Bash like in this case `${key_file}.tmp` → `"${key_file}.tmp"` in the future or I will have to do that when I rebase :wink:

**Comment by drybjed** [comment]
@ypid OK! I guess it's time to install [shellcheck](http://www.shellcheck.net/), or better yet, add it as an option in the travis tests, so you can specify files to check with it.

**Comment by ypid** [comment]
>  OK! I guess it's time to install shellcheck, or better yet, add it as an option in the travis tests, so you can specify files to check with it.

:+1: Exactly what I was thinking :wink: Note that there is [syntastic for Vim](https://github.com/scrooloose/syntastic) which integrates shellcheck with Vim.

**Comment by drybjed** [comment]
Thanks, I'll check it out.

**Pull request merged**